events/models.py

```diff
@@ -29,6 +29,7 @@ class Event(models.Model):
         default=True,
         help_text="If this is checked all users that attends the event needs to \
         be accepted by an admin.")
+    external_signup_url = models.CharField(max_length=80, blank=True)
```

Comment:
It says that an URLField is implemented as a CharField but with a default of 200 chars, so I don't think there's any difference really other than that :) Maybe it does some sort of basic validation, no idea.